Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,103 @@
+    - This is a roleplaying game, thus IC conflicts must only be resolved with IC (Local) chat. Utilizing voice chat, calling across the room, OOC, or other out-of-game communication methods to coordinate in-game actions is forbidden and will result in a [color=#ff0000]appeal-only permanent ban[/color] in most cases.
+    - Using emotes or LOOC to bypass IC chat restrictions (accents, unconsciousness, death) is not allowed.
+    - Preemptive spacing or destruction of high-risk items also falls under this rule.
+
+Security Rules
+
+[color=#a4885c]12.[/color] Security is expected to use non-lethal measures unless faced with lethal force or a situation where non-lethal measures aren’t possible. Ideally, do not enter combat mode, as security stun batons do not deal damage, but still stun, when outside of combat mode.
+    - If you still manage to kill someone as security, get them cloned immediately, and make sure their stuff is returned. Security should make their best effort to keep criminals alive and in custody.
+[color=#a4885c]13.[/color] Prisoners should not be given sentences longer than 10 minutes. Certain extreme exceptions like bombers and assassins may warrant permabrigging, or in the very worst cases, execution.
+    - Note that employment at a hostile corporation (i.e. traitor status) alone does not qualify someone for perma or execution.
+    - Security are required to follow Space Law. If in doubt, consult Centcom’s law team.
+
+MRP Specific Rules
+
+[color=#a4885c]14.[/color] This server is a Roleplay Expected server. We define this as performing your assigned duties. This is a ‘do your job’ server. This means it is important to do what is expected out of your department and not what would be expected out of other departments.
+    - Don’t do other peoples jobs for them. Opt in for the role you intend to play or change your job IC. Failure to do your basic duties may result in a job ban.
+
+[color=#a4885c]15.[/color] Each round exists independently of all other rounds. While friendships persisting through rounds is okay, using experiences from other rounds to give an advantage or disadvantage to other players is not permitted.
+
+[color=#a4885c]16.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living.
+
+Note: SSD characters are players who are disconnected or AFK. It is possible for players to return to SSD bodies. Players become catatonic when they take a ghost spawner role or commit suicide. Players are NOT able to return to these bodies without admin intervention.
+
+[color=#a4885c]17.[/color] Do your best not to interact negatively with SSD/AFK players. Moving them to safety is acceptable, but killing, looting, or otherwise griefing them while they are away is not.
```

Comment:
```suggestion
[color=#a4885c]17.[/color] Do your best not to interact negatively with SSD/AFK players. Moving them to safety is acceptable; killing, looting, or otherwise griefing them while they are away is not.
```